unified_targets/configs/MANGOF405.config

```diff
@@ -0,0 +1,126 @@
+# Betaflight / MANGOF405 (MGF4) 4.0.3 Jun 26 2019 / 10:17:27 () MSP API: 1.41
```

Comment:
Should be:
`# Betaflight / STM32F405 (S405) 4.0.3 Jun 26 2019 / 10:17:27 () MSP API: 1.41`